# TASK_1:
#   EDGE: [1, 1]
#   TYPE: "go_to_goal_predicate_2d"
#   CENTER: [0.0, 4.0]
#   EPSILON: 1.0
#   TEMP_OP: "AlwaysOperator"
#   INTERVAL: [70, 100]
#   INVOLVED_AGENTS: [1]
#   START_TIME: 0.0

# TASK_2:
#   EDGE: [1, 2]
#   TYPE: "formation_predicate"
#   CENTER: [2.0, 2.0]
#   EPSILON: 2.0
#   TEMP_OP: "AlwaysOperator"
#   INTERVAL: [70, 100]
#   INVOLVED_AGENTS: [1, 2]
#   START_TIME: 0.0

# TASK_3:
#   EDGE: [4, 4]
#   TYPE: "go_to_goal_predicate_2d"
#   CENTER: [2.0, 6.0]
#   EPSILON: 1.0
#   TEMP_OP: "AlwaysOperator"
#   INTERVAL: [90, 100]
#   INVOLVED_AGENTS: [4]
#   START_TIME: 0.0
  
# TASK_4:
#   EDGE: [2, 4]
#   TYPE: "epsilon_position_closeness_predicate"
#   CENTER: [0.0, 0.0]
#   EPSILON: 4.0
#   TEMP_OP: "AlwaysOperator"
#   INTERVAL: [70, 100]
#   INVOLVED_AGENTS: [2, 4]
#   START_TIME: 0.0

# TASK_5:
#   EDGE: [1, 3]
#   TYPE: "formation_predicate"
#   CENTER: [-2.0, 2.0]
#   EPSILON: 2.0
#   TEMP_OP: "AlwaysOperator"
#   INTERVAL: [80, 100]
#   INVOLVED_AGENTS: [1, 3]
#   START_TIME: 0.0

# TASK_6:
#   EDGE: [3, 5]
#   TYPE: "formation_predicate"
#   CENTER: [0.0, 3.0]
#   EPSILON: 2.0
#   TEMP_OP: "AlwaysOperator"
#   INTERVAL: [90, 100]
#   INVOLVED_AGENTS: [3, 5]
#   START_TIME: 0.0

TASK_1:
  EDGE: [1, 2]
  TYPE: "epsilon_position_closeness_predicate"
  CENTER: [0.0, 0.0]
  EPSILON: 2.0
  TEMP_OP: "AlwaysOperator"
  INTERVAL: [30, 50]
  INVOLVED_AGENTS: [1, 2]
  START_TIME: 0.0

TASK_2:
  EDGE: [2, 4]
  TYPE: "epsilon_position_closeness_predicate"
  CENTER: [0.0, 0.0]
  EPSILON: 3.0
  TEMP_OP: "AlwaysOperator"
  INTERVAL: [40, 50]
  INVOLVED_AGENTS: [2, 4]
  START_TIME: 0.0

TASK_3:
  EDGE: [1, 3]
  TYPE: "epsilon_position_closeness_predicate"
  CENTER: [-2.0, 2.0]
  EPSILON: 2.0
  TEMP_OP: "AlwaysOperator"
  INTERVAL: [30, 50]
  INVOLVED_AGENTS: [1, 3]
  START_TIME: 0.0

TASK_4:
  EDGE: [3, 5]
  TYPE: "formation_predicate"
  CENTER: [1.0, 3.0]
  EPSILON: 2.0
  TEMP_OP: "AlwaysOperator"
  INTERVAL: [40, 50]
  INVOLVED_AGENTS: [3, 5]
  START_TIME: 0.0

TASK_5:
  EDGE: [4, 4]
  TYPE: "go_to_goal_predicate_2d"
  CENTER: [0.0, 2.0]
  EPSILON: 1.0
  TEMP_OP: "AlwaysOperator"
  INTERVAL: [40, 50]
  INVOLVED_AGENTS: [4]
  START_TIME: 0.0

# # ---- Phase 2 ----

TASK_6:
  EDGE: [1, 1]
  TYPE: "go_to_goal_predicate_2d"
  CENTER: [0.0, 4.0]
  EPSILON: 2.0
  TEMP_OP: "AlwaysOperator"
  INTERVAL: [170, 200]
  INVOLVED_AGENTS: [1]
  START_TIME: 0.0

TASK_7:
  EDGE: [1, 2]
  TYPE: "formation_predicate"
  CENTER: [2.0, 2.0]
  EPSILON: 3.0
  TEMP_OP: "AlwaysOperator"
  INTERVAL: [170, 200]
  INVOLVED_AGENTS: [1, 2]
  START_TIME: 0.0

TASK_8:
  EDGE: [4, 4]
  TYPE: "go_to_goal_predicate_2d"
  CENTER: [1.0, 5.0]
  EPSILON: 2.0
  TEMP_OP: "AlwaysOperator"
  INTERVAL: [190, 200]
  INVOLVED_AGENTS: [4]
  START_TIME: 0.0
  
TASK_9:
  EDGE: [2, 4]
  TYPE: "epsilon_position_closeness_predicate"
  CENTER: [0.0, 0.0]
  EPSILON: 4.0
  TEMP_OP: "AlwaysOperator"
  INTERVAL: [170, 200]
  INVOLVED_AGENTS: [2, 4]
  START_TIME: 0.0

TASK_10:
  EDGE: [1, 3]
  TYPE: "formation_predicate"
  CENTER: [-1.0, 1.0]
  EPSILON: 2.0
  TEMP_OP: "AlwaysOperator"
  INTERVAL: [180, 200]
  INVOLVED_AGENTS: [1, 3]
  START_TIME: 0.0

TASK_11:
  EDGE: [3, 5]
  TYPE: "formation_predicate"
  CENTER: [0.0, 3.0]
  EPSILON: 2.0
  TEMP_OP: "AlwaysOperator"
  INTERVAL: [190, 200]
  INVOLVED_AGENTS: [3, 5]
  START_TIME: 0.0
